app/controllers/stories/feeds_controller.rb

```diff
@@ -46,7 +47,7 @@ def signed_out_base_feed
         Articles::Feeds::Basic.new(user: nil, page: @page, tag: params[:tag]).feed
       else
         Articles::Feeds::LargeForemExperimental.new(user: current_user, page: @page, tag: params[:tag])
-          .default_home_feed(user_signed_in: user_signed_in?)
+          .default_home_feed(user_signed_in: false)
```

Comment:
Given that this is the `signed_out_base_feed` method and we already performed the `user_signed_in?` check-in `assign_feed_stories` to even get to this point we can hard-code the value to `false`.